apps/nowcasting-app/components/hooks/use-hot-key-control-chart.tsx

```diff
@@ -14,13 +14,13 @@ const useHotKeyControlChart = (limits?: { start: string; end: string }) => {
             formatISODateString(selectedISOTime || "") === formatISODateString(limits?.start || "")
           )
             return selectedISOTime;
-          return addMinutesToISODate(selectedISOTime || "", -30);
+          return addMinutesToISODate(selectedISOTime || "", -15);
```

Comment:
@rachel-labri-tipton did you get to the bottom of why the hotkeys were skipping an hour instead of the 30mins? Sounds like could be two instances of this function firing if this is the fix ☝️ 🤔